src/runtime/runtime.ts

```diff
@@ -35,16 +36,19 @@ export type RuntimeArcOptions = Readonly<{
   inspectorFactory?: ArcInspectorFactory;
 }>;
 
+let runtime: Runtime = null;
```

Comment:
`let runtime: Runtime? = null`

Reply:
What does that do?

Reply:
Shouldn't it be `let runtime: Runtime | null = null;`?

It's not clear from the docs (so far) if `?` handles `null` as well as `undefined`.

Reply:
Yes, that's a bit stricter, good call.